great_expectations/validator/validator.py

```diff
@@ -1978,6 +1964,9 @@ def validate(
                 configurations=expectations_to_evaluate,
                 runtime_configuration=runtime_configuration,
             )
+            if self._include_rendered_content:
+                for validation_result in results:
+                    validation_result.render()
```

Comment:
`Validator.validate` is used by `Checkpoint.run` so this enables rendering when only a `Checkpoint` is explicitly defined.